hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieMetadataTableValidator.java

```diff
@@ -180,6 +181,8 @@ public class HoodieMetadataTableValidator implements Serializable {
 
   private final String taskLabels;
 
+  private List<Throwable> throwables = new LinkedList<>();
```

Comment:
Nitpick: Any reason to use linked list over array list?

Reply:
Addressed. No particular preference in this case. Generally I tend to use linked list if the size of list is not known and I don't need random access.